apps/nuxt3-ssr/components/VariableCard.vue

```diff
@@ -0,0 +1,42 @@
+<script setup lang="ts">
```

Comment:
I think we should try to make the variable card so usefull that people will not feel the need to drill down, except for wanting to look at harmonisation details.

Then I suppose we need (format, description, unit and optionally permitted values)

<img width="978" alt="image" src="https://github.com/molgenis/molgenis-emx2/assets/120060/0c663841-515b-4e38-b6ff-f9ef06c00cdd">

Reply:
Harmonization is still a 'stub' right?

Reply:
yes , its a stub , the whole variable / network section is behind a feature flag